src/vl-map-layer-style.js

```diff
@@ -55,7 +67,52 @@ export class VlMapLayerStyle extends vlElement(HTMLElement) {
    * @Return {string}
    */
   get textColor() {
-    return this.getAttribute('text-color') || '#FFF';
+    return this.getAttribute('text-color') || 'rgba(0, 0, 0, 1)';
+  }
+
+  /**
+   * Geeft de tekstkleur van de rand van de stijl terug.
+   *
+   * @Return {string}
+   */
+  get textBackgroundColor() {
+    return this.getAttribute('text-background-color') || 'rgba(0, 0, 0, 0)';
+  }
+
+  /**
+   * Geeft de tekstkleur van de rand van de stijl terug.
+   *
+   * @Return {string}
+   */
+  get textBorderColor() {
+    return this.getAttribute('text-border-color') || 'rgba(255, 255, 255, 1)';
+  }
```

Comment:
Ik zou default 0 transparantie geven in combinatie met de default blauwe achtergrondkleur. De default settings zorgen nu voor een label dat zeer moeilijk leesbaar is.